constantdelay.go

```diff
@@ -22,6 +22,14 @@ func Every(duration time.Duration) ConstantDelaySchedule {
 
 // Next returns the next time this should be run.
 // This rounds so that the next activation time will be on the second.
-func (schedule ConstantDelaySchedule) Next(t time.Time) time.Time {
-	return t.Add(schedule.Delay - time.Duration(t.Nanosecond())*time.Nanosecond)
+// If `after` time is specified then the next activation time later than
+// `after` is calculated.
```

Comment:
Even though it is a short method, can you add documentation, on each parameter?

Reply:
Updated comments.